.nvmrc

```
@@ -1 +1 @@
-lts/gallium
+v18
```

Any reason not to just set this as 
```suggestion
18.13.0
```

I had an older version of 18, so when I ran `nvm use` then `nvm install` it gave me warnings. Not a big deal, but took me a second to figure out what was going on. My worry is this might cause some confusion for others too. Any harm in just being specific throughout?